learn/tanssi/overview.md

```diff
@@ -5,7 +5,7 @@ description: Tanssi is an Appchain protocol that eases the process of deploying
 
 # What is Tanssi? {: #what-is-tanssi }
 
-Tanssi is an Appchain infrastructure protocol that makes deploying blockchains with custom logic specific to certain types of applications significantly easier than what is available today. These customized blockchains are normally referred to as Appchains, but in Tanssi terms, they are also known as ContainerChains.
+Tanssi is an Appchain infrastructure protocol that makes deploying blockchains with custom logic specific to certain types of applications significantly easier than what is available today. These customized blockchains are normally referred to as Appchains, but in Tanssi terms, they are also known as Appchains or ContainerChains.
```

Comment:
```suggestion
Tanssi is an Appchain infrastructure protocol that makes deploying blockchains with custom logic specific to certain types of applications significantly easier than what is available today.
```

I don't think we need to even mention this, but let me know if you think otherwise!